### DISCUSSION TITLE
How to make StackOverflow like tag selector

### DISCUSSION THREAD

**Discussion by piszczu4** [JedWatson/react-select#5822] [Q&A] [answered]
Hi, SO has the following tag selector where we have multiple tags in one row and additionally there is a description. Is it possible to reproduce it using `react-select`?
![image](https://github.com/JedWatson/react-select/assets/88668060/e43c79a4-0819-47b8-a0ff-8048aa46bbf1)

**Comment by manjushsh** [comment, marked as answer]
Hi @piszczu4, you can achieve this with help of custom components and guide for layout as suggested by @ebonow  [here](https://github.com/JedWatson/react-select/discussions/4341) and custom filters etc.

Layout [example codesandbox](https://codesandbox.io/p/sandbox/react-select-grid-menu-options-ye2b8?file=%2Findex.js)
